build/build-image/cross/Makefile

```diff
@@ -14,15 +14,18 @@
 
 .PHONY:	build push
 
-REGISTRY?=staging-k8s.gcr.io
+REGISTRY?=gcr.io/k8s-staging-build-image
```

Comment:
Does not match what's in Dockerfile?

Reply:
@dims -- which Dockerfile are you referring to here?
That's meant to be a var for the staging registry. The other image (`build/build-image/Dockerfile`) pulls from the prod registry.

Reply:
```
STAGING_REGISTRY?=gcr.io/k8s-staging-build-image
PROD_REGISTRY?=us.gcr.io/k8s-artifacts-prod/build-image
IMAGE=kube-cross
```

and then in the build step:
```
	docker build \
		-t $(STAGING_REGISTRY)/$(IMAGE):$(TAG) \
		-t $(PROD_REGISTRY)/$(IMAGE):$(KUBE_CROSS_VERSION) \
		.
```

That way, you can build and test it locally, but the push step will still push to staging (should a manual push be needed)